completions/ip

```diff
@@ -417,10 +417,14 @@ _comp_cmd_ip()
                     ;;
                 *)
                     ((cword == subcword)) &&
-                        _comp_compgen -- -W 'state policy monitor'
+                        _comp_compgen -- -W 'state policy monitor help'
```

Comment:
The other cases seem to have `help` at the beginning of the list.

Reply:
Changed it, also in netconf